monosketch-svelte/src/lib/ui/modal/common/NoBackgroundModal.svelte

```diff
@@ -7,6 +7,7 @@ dismiss the modal correctly.
 <script lang="ts">
 import { onMount } from 'svelte';
 
+export let onSubmit: () => void = () => {};
```

Comment:
I’ve initialized it as an empty function to ensure it's optional—other components using this one won’t need any changes, and everything will work as before if the function isn’t needed.